sdks/java/io/clickhouse/src/test/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIOTest.java

```diff
@@ -391,22 +391,22 @@ public void testArrayOfPrimitiveTypes() throws Exception {
     try (ResultSet rs = executeQuery("SELECT * FROM test_array_of_primitive_types")) {
       rs.next();
 
-      assertEquals("['2030-10-01','2031-10-01']", rs.getString("f0"));
-      assertEquals("['2030-10-09 08:07:06','2031-10-09 08:07:06']", rs.getString("f1"));
-      assertEquals("[2.2,3.3]", rs.getString("f2"));
-      assertEquals("[3.3,4.4]", rs.getString("f3"));
-      assertEquals("[4,5]", rs.getString("f4"));
-      assertEquals("[5,6]", rs.getString("f5"));
-      assertEquals("[6,7]", rs.getString("f6"));
-      assertEquals("[7,8]", rs.getString("f7"));
-      assertEquals("['eight','nine']", rs.getString("f8"));
-      assertEquals("[9,10]", rs.getString("f9"));
-      assertEquals("[10,11]", rs.getString("f10"));
-      assertEquals("[11,12]", rs.getString("f11"));
-      assertEquals("[12,13]", rs.getString("f12"));
-      assertEquals("['abc','cde']", rs.getString("f13"));
-      assertEquals("['cde','abc']", rs.getString("f14"));
-      assertEquals("[true,false]", rs.getString("f15"));
+      assertEquals("[2030-10-01, 2031-10-01]", rs.getString("f0"));
```

Comment:
Looks like all the string fields no longer have single quotes around them. Is that the breaking change introduced by the clickhouse-server version upgrade?

Reply:
After I resolve the connection issue I already see these breaking changes without bump clickhouse/clickhouse-server and zookeeper docker version.

That said the different string is due to testcontainer 1.16 -> 1.19 version change, however which exact component of test container caused this is not known